lib/publiccloud/azure_client.pm

```diff
@@ -40,7 +40,7 @@ sub init {
 
 sub az_login {
     my ($self) = @_;
-    my $login_cmd = sprintf(q(while ! az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET -t $ARM_TENANT_ID; do sleep 10; done),
```

Comment:
The output here isn't read, so stderr could be preserved?

Reply:
Ah yes you are correct. Sorry, my mistake. I missed there is another message in the log file. Will revert this change and re-run the VR.

Reply:
Fixed and VR passed as well.